src/test/java/nl/tudelft/pixelperfect/event/EventParametersTest.java

```diff
@@ -111,6 +122,14 @@ public void testEqualsDifferentSizes() {
   }
 
   /**
+   * An EventParameters instance should no be considered equal to something of another type.
```

Comment:
You want to test whether something is not equal to another type, but you only test null, which would never be true. Maybe just test it against a random other class.